packages/introspect/src/e2e.test.ts

```diff
@@ -81,4 +99,74 @@ describe('e2e', () => {
                 }),
         });
     }, 100_000);
+
+    test('Add to Pagila', async () => {
+        await deleteData();
+
+        await createTable();
+
+        await insertData();
+
+        for (const [name, value] of Object.entries(await selectData())) {
+            console.log({
+                'Name:': name,
+                'Value:': value,
+                'JS primitive type': typeof value,
+            });
+        }
+
+        await deleteData();
+    }, 100_000);
 });
+
+const createTable = async () => {
+    try {
+        await client.connect();
```

Comment:
You should re-use the same client between all these calls. No need to re-connect on every statement.